cli/tox.ini

```diff
@@ -23,11 +23,11 @@ commands =
 
 [testenv:py27-integration]
 commands =
-  py.test -vv {env:CI_FLAGS:} tests/integrations{posargs}
+  py.test -vv -x {env:CI_FLAGS:} tests/integrations{posargs}
 
 [testenv:py34-integration]
 commands =
-  py.test -vv {env:CI_FLAGS:} tests/integrations{posargs}
+  py.test -vv -x {env:CI_FLAGS:} tests/integrations{posargs}
```

Comment:
@mgummelt and @tamarrow, I know that you both talked about this but I found it very helpful while testing this change locally. What do you we think?